app/src/organisms/ProtocolSetupParameters/index.tsx

```diff
@@ -225,26 +276,19 @@ export function ProtocolSetupParameters({
       >
         {sortRuntimeParameters(runTimeParametersOverrides).map(
           (parameter, index) => {
-            const detailLabelForCsv =
-              mostRecentAnalysis?.result === 'parameter-value-required'
-                ? t('required')
-                : parameter.displayName
-
-            let setupStatus: 'ready' | 'not ready' | 'general' | 'inform' =
-              'inform'
-            if (
-              enableCsvFile &&
-              parameter.type === 'csv_file' &&
-              mostRecentAnalysis?.result === 'parameter-value-required'
-            ) {
-              setupStatus = 'not ready'
-            }
-            if (
-              enableCsvFile &&
-              parameter.type === 'csv_file' &&
-              mostRecentAnalysis?.result === 'ok'
-            ) {
-              setupStatus = 'ready'
+            let detail: string = ''
+            let setupStatus: ProtocolSetupStepStatus
+            if (parameter.type === 'csv_file') {
```

Comment:
For safe, still we want to keep `enableCsvFile`.

```suggestion
            if (enableCsvFile && parameter.type === 'csv_file') {
```